.pre-commit-config.yaml

```
@@ -52,3 +53,9 @@ repos:
     hooks:
       - id: cython-lint
         args: [--no-pycodestyle]
+  - repo: https://github.com/sphinx-contrib/sphinx-lint
+    rev: v0.9.1
+    hooks:
+      - id: sphinx-lint
+        files: ^docs/
+        args: ['--disable', 'all', '--enable', 'trailing-whitespace,missing-final-newline', 'docs']
```

Can we remove `docs` from this list? (I think that duplicated logs are caused by this.)

Could you use multi-lines array style instead of one-line array style? It seems that we need to specify many command line options later. One-line array style will be difficult to understand.

```suggestion
        args:
          - --disable
          - all
          - --enable
          - trailing-whitespace,missing-final-newline
```

---

Nice change, thanks. Accepted.

I'm not sure about the last but but maybe we don't. Good eye.